### ISSUE TITLE
Support for Ubuntu 21.10 Impish Indri

### ISSUE THREAD

**Issue by LouisMT** [docker/for-linux#1305]
Please add support for Ubuntu 21.10. It will be release on October 14.

Also see #940 and #832. I'm opening the new issue because the first one seems a bit stale.

Thanks.

**Comment by noahehall** [comment]
+1 

currently getting `The repository 'https://download.docker.com/linux/ubuntu impish Release' does not have a Release file.`

**Comment by thaJeztah** [comment]
> Before merging docker/docker-ce-packaging#585, they will probably need to upgrade their CI/CD infrastructure to Docker v20.10.10 or later, because Dockerfile with FROM ubuntu:21.10 cannot be built with Docker prior to v20.10.10.

Yes, this is a bit of a chicken-and-egg situation; I started a test-build for 20.10.10, and if all goes well, planning to publish packages for an `rc1` to our package repositories, and update our CI nodes to use that.
